cassiopeia/plotting/itol_utilities.py

```diff
@@ -45,7 +45,9 @@ def upload_and_export_itol(
     use_branch_lengths: bool = False,
     palette: List[str] = palettes.Category20[20],
     random_state: Optional[np.random.RandomState] = None,
+    files: Optional[List[str]] = None,
```

Comment:
Let's name this something more informative, like `user_plotting_files`. Happy to entertain better names, but `files` is too obscure, I think.